dev/app.js

```diff
@@ -122,6 +122,8 @@ search.addWidget(
 
 search.addWidget(
   instantsearch.widgets.refinementList({
+    collapsable: true,
+    collapsed: true,
```

Comment:
Ultimately I would have wanted to have only a single configuration entry for this option, like we have for showMore: {limit..}.

But could not think of any good one.

Do you have any idea?